st2actions/st2actions/cmd/actionrunner.py

```diff
@@ -19,9 +20,22 @@
     time=True)
 
 
+def _setup_sigterm_handler():
```

Comment:
Its possible. It will be easier to implement a Base type if a pattern emerges. For now it will be just 1 method that registers handlers.

There is already https://github.com/StackStorm/st2/blob/master/st2common/st2common/service_setup.py